apps/chat/src/utils/auth/auth-callbacks.ts

```diff
@@ -37,9 +38,11 @@ const getUser = (accessToken: string | undefined, providerId: string) => {
   const roles = Array.isArray(dialRoles) ? dialRoles : [dialRoles];
   const isAdmin =
     roles.length > 0 && adminRoleNames.some((role) => roles.includes(role));
+  const codeAppRoles = process.env.CODE_APPS_ROLES?.split(',') ?? [];
 
   return {
     isAdmin,
+    canCreateCodeApps: !!intersection(codeAppRoles, roles).length,
```

Comment:
allow if CODE_APPS_ROLES isn't configured, but CodeApps are enabled
```suggestion
    canCreateCodeApps: !codeAppRoles.length || !!intersection(codeAppRoles, roles).length,
```

Reply:
line 41 has default value. Are you sure that this change is necessary?